shell-completion/hledger-completion.bash.m4

```diff
@@ -1,102 +1,102 @@
+# shellcheck disable=2034
 
 # Completion script for hledger.
 # Created using a Makefile and real hledger.
 
-# No set -e because this file is sourced and is not supposed to quit the current shell.
-set -o pipefail
+# This script is sourced by an interactive shell, so do NOT do things like
```

Comment:
Good catch, I probably forgot that this file is sourced...

Reply:
This one is kinda self-contained, so I'll mark it as resolved.